### DISCUSSION TITLE
Multiple tab for same form

### DISCUSSION THREAD

**Discussion by AaronRiboy** [serenity-is/Serenity#6639] [Help] [answered]
<img width="686" alt="image" src="https://user-images.githubusercontent.com/29535650/210700870-674f485a-b7b7-4440-b1af-1bbc22d9f153.png">
Guys, where i can refer to implement this multiple tabs. Serenity tutorial documentation doenst have some of this basic stuff. I tried to refer  at northwind but its getting confused .Im still learning this framework.

**Comment by edwardch** [comment, marked as answer]
Have a look in the `common-features` project, then in `src/Serenity.Demo.Northwind/Modules/Customer/CustomerDialog.ts`.

The getTemplate section shows how to add tabs (you can recognise the html).

There's another file in there named CustomerOrdersGrid.ts, that has a class which the dialog file refers to. 

Hth

**Reply by AaronRiboy** [reply]
Thank you for the fast response .
Tried to pull the repo and run .Got some error to run
<img width="735" alt="image" src="https://user-images.githubusercontent.com/29535650/210713617-5b55f46a-63c6-4dbb-a954-b5c1993fd2d1.png">

https://github.com/serenity-is/common-features

**Reply by edwardch** [reply]
I don't think you can run it like that, but rather use the code in those folders as an example...
